data/bookkeeping/block_test.go

```diff
@@ -832,9 +832,9 @@ func TestBlock_ContentsMatchHeader(t *testing.T) {
 	copy(block.BlockHeader.TxnCommitments.Sha256Commitment[:], rootSliceSHA256)
 	a.False(block.ContentsMatchHeader())
 
-	/* Test Consensus Future */
+	/* Test Consensus Current */
 	// Create a block with SHA256 TxnCommitments
-	block.CurrentProtocol = protocol.ConsensusFuture
+	block.CurrentProtocol = protocol.ConsensusCurrentVersion
```

Comment:
Should this be fixed to v39?

I'm not sure what others think, but I'm generally against testing with the current consensus version for two reasons:
1. If you want to always test behavior on the latest version, using vFuture avoids surprises the next time we assign future to current. It's much better to be informed of a test failure during development of new vFuture behavior rather than at the time of release.
2. If you want to test behavior only in the version we enable a consensus param or across version boundaries, it's better to test against that specific consensus version.

Reply:
Oh, this is a good point. 🤔

Reply:
So I made this change - most things test against current consensus version, I'm open to changing it back but this is really an outlier at this point vs the rest of the codebase.

Reply:
Right, fair enough, a lot of the codebase does test with current consensus, so we probably ought to have a separate discussion about it. That shouldn't block this PR.